services/src/main/scala/cromwell/services/metadata/package.scala

```diff
@@ -1,5 +1,12 @@
 package cromwell.services
 
+import cromwell.services.metadata.MetadataService.{MetadataReadAction, MetadataServiceResponse}
+import spray.json.JsObject
+
 package object metadata {
   type QueryParameters = Seq[QueryParameter]
 }
+
+sealed trait BuildMetadataResponse extends MetadataServiceResponse { def originalRequest: MetadataReadAction }
```

Comment:
Even by my standards this name is terrible, suggestions very welcome.

Reply:
`BuildMetadataJsonResponse`?

Reply:
or just `MetadataJsonResponse`?